packages/frontend/src/components/file-upload-area/file-upload-area.tsx

```diff
@@ -38,7 +39,13 @@ const defaultDragElement = (
   </Box>
 );
 
-export const FileUploadArea = ({ onDrop, accept = DROPZONE_ACCEPT_ALL_FILES, element, dragElement }: Props) => {
+export const FileUploadArea = ({
+  onDrop,
+  accept = DROPZONE_ACCEPT_ALL_FILES,
+  element,
+  dragElement,
+  display
+}: Props) => {
```

Comment:
Typically what I'd do here is use BoxProps instead of hardcoding a specific property like `display`:

```suggestion
  ...boxProps
}: Props & BoxProps) => {
```